common4j/src/main/com/microsoft/identity/common/java/controllers/ExceptionAdapter.java

```diff
@@ -92,7 +92,7 @@ public static BaseException exceptionFromAuthorizationResult(@NonNull  final Aut
                     methodTag,
                     "AuthorizationErrorResponse is not set"
             );
-            return new ClientException(ClientException.UNKNOWN_ERROR, "Authorization failed with unknown error. Authorization Status: " +authorizationResult.getAuthorizationStatus());
+            return new ClientException(ClientException.AUTHORIZATION_RESULT_ERROR, "Authorization failed with unknown error. Authorization Status: " +authorizationResult.getAuthorizationStatus());
```

Comment:
We can probably rename the error code here to `AUTHORIZATION_RESULT_NULL_ERROR_RESPONSE`

Reply:
+1 (and also improve the error message accordingly?)

Reply:
Addressed here: 96ab98a6d06c248fb699749acea717d36f61c015